src/orchestrator/tools.py

```
@@ -1,36 +1,299 @@
 """Define he agent's tools."""
 
+from collections.abc import Coroutine
```

Ruff (0.11.9)

`collections.abc.Coroutine` imported but unused

Remove unused import: `collections.abc.Coroutine`

(F401)